app/views/comparisons/change_in_electricity_holiday_consumption_previous_years_holiday/_tables.html.erb

```diff
@@ -0,0 +1,100 @@
+<table id="comparison-table" class="table advice-table table-sorted">
+  <thead>
+    <tr>
+      <th><%= t('analytics.benchmarking.configuration.column_headings.school') %></th>
+      <th class="text-right">
+        <%= t('analytics.benchmarking.configuration.column_headings.change_pct') %>
+      </th>
+      <th class="text-right">
+        <%= t('analytics.benchmarking.configuration.column_headings.change_£current') %>
+      </th>
+      <th class="text-right">
+        <%= t('analytics.benchmarking.configuration.column_headings.change_kwh') %>
+      </th>
+      <th class="text-right">
+        <%= t('analytics.benchmarking.configuration.column_headings.most_recent_holiday') %>
+      </th>
+      <th class="text-right">
+        <%= t('analytics.benchmarking.configuration.column_headings.previous_holiday') %>
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <% pupils_changed_footnote = false %>
+    <% tariff_has_changed_footnote = false %>
+    <% infinite_increase_footnote = false %>
+    <% infinite_decrease_footnote = false %>
+    <% results.each do |result| %>
+      <tr>
+        <td>
+          <%= result.school.name %>
+          <% if result.pupils_changed %>
+            <% pupils_changed_footnote = true %>
+            <a href="#footnotes">(*1)</a>
+          <% end %>
+          <% if result.difference_percent.infinite? == 1 %>
+            <% infinite_increase_footnote = true %>
+            <a href="#footnotes">(*2)</a>
+          <% end %>
+          <% if result.difference_percent.infinite? == -1 %>
+            <% infinite_decrease_footnote = true %>
+            <a href="#footnotes">(*3)</a>
+          <% end %>
+          <% if result.tariff_has_changed %>
+            <% tariff_has_changed_footnote = true %>
+            <a href="#footnotes">(*6)</a>
+          <% end %>
+        </td>
+        <td class="text-right">
+          <%= up_downify(format_unit(result.difference_percent, :relative_percent_0dp, true, :benchmark)) %>
+        </td>
+        <td class="text-right">
+          <%= up_downify(format_unit(result.difference_gbpcurrent, :£_0dp, true, :benchmark)) %>
+        </td>
+        <td class="text-right">
+          <%= up_downify(format_unit(result.difference_kwh, :kw, true, :benchmark)) %>
+        </td>
+        <td class="text-right">
+          <%= result.name_of_current_period %>
```

Comment:
We're going to need to make some changes to the alert class here for `name_of_current_period` and `name_of_previous_period`. These values will be text and we are only storing values that have been generated with locale of English. So Welsh uses will see English period names.

The `period_name` code looks something like:

```
  protected def period_name(period)
    I18nHelper.holiday(period.type)
  end
```

So we'll need to change the alert to declare a variable and add a method that returns the `period.type`. We can then we can use that to display the correct text in the table.

We only need to do this for any text to be displayed. Any constants can be used as is.

Reply:
I've not looked at the values recently. If they can be easily mapped back to existing keys that might be simpler, but if we can fix at source that's probably for the best.